Tella/Scenes/Settings/Views/Servers/ServersListView.swift

```diff
@@ -74,15 +75,21 @@ struct ServersListView: View {
         }
     }
     
-    private func handleActions(item: ListActionSheetItem) {
+    private func handleActions(item: ListActionSheetItem, server: Server) {
         guard let type = item.type as? ServerActionType else { return  }
         
         switch type {
         case .edit:
+            if (server.serverType == ServerConnectionType.uwazi.rawValue) {
+                navigateTo(destination: UwaziAddServerURLView(appModel: mainAppModel, server: server)
```

Comment:
I think edit Uwazi server should show edit name and not show add server URL 
see https://www.figma.com/file/7CWTHcevVxO4Rb1NLRF74O/Tella-2.0-prototype?type=design&node-id=8237-345020&mode=design&t=s6q8VZrZHgmGcWtK-0